LLIO2025/src/services/TaskService.ts

```
@@ -0,0 +1,32 @@
+            const taskToUpdate = {
+                ...task,
+                startDateTime: new Date(task.startDateTime),
+                endDateTime: new Date(task.endDateTime)
+            };
+            return await TaskApiService.updateTask(taskToUpdate);
+        } catch (error) {
+            console.error("Erreur lors de la mise à jour de la tâche", error);
+            throw error;
+        }
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)